pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
```

Comment:
I guess I need to revert that, right?

this was indroduced after running `pnpm remove rimraf`
```suggestion
```

Reply:
Eh, this is probably just pnpm doing what pnpm needs to do. I think this was s difference back on v6 vs current v8?

Reply:
probably. I'm using v8